modules/integration_azure_firewall/conf/02-throughput.yaml

```diff
@@ -0,0 +1,27 @@
+## Example
+module: Azure firewall
+name: throughput
+
+transformation: true
+aggregation: true
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+
+tip: "Firewall will autoscale at 2.5Gbps but the maximum throughput is 30Gbps"
+
+signals:
+  signal:
+    metric: "Throughput"
+rules:
+  critical:
+    threshold: 4026531840
+    comparator: ">"
+  major:
+    threshold: 3355443200
+    comparator: ">"
+    lasting_duration: "900s"
```

Comment:
specific thresholds values and lasting for only one alert should be documented as notes.

it could be good to also add the unit. speaking of that, you can optionally use a `scale()` function to convert to a unit more human readable (MB/Gb).